### PR TITLE
[11.0] [MIG] Migrating "sale_mrp_link" to 11.0

### PR DESCRIPTION
**Pull request by NachoAlesLopez**
@guadaltech

### CODE REVIEW

File: sale_mrp_link/README.rst [hunk 1]
```diff
@@ -14,13 +14,13 @@ Sale MRP Link
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
-    :target: https://github.com/OCA/sale-workflow/tree/10.0/sale_mrp_link
+    :target: https://github.com/OCA/sale-workflow/tree/12.0/sale_mrp_link
```
Comment (nikul-serpentcs) [inline sale_mrp_link/README.rst]: @NachoAlesLopez 11.0, not 12.0
Everywhere

Comment (NachoAlesLopez) [reply sale_mrp_link/README.rst]: Done!

**Review by mariaasunciondaza — APPROVED** [review]
Functional and code review Ok

File: sale_mrp_link/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Alex Comba - Agile Business Group
+# Copyright 2016-2018 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Sale MRP Link',
+    'summary': 'Show manufacturing orders generated from sales order',
+    'version': '11.0.1.0.0',
+    'development_status': 'Production/Stable',
+    'category': 'Sales Management',
```
Comment (rousseldenis) [inline sale_mrp_link/__manifest__.py]: Replace this by 'Sales'

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comment

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

File: sale_mrp_link/__manifest__.py (hunk 2 above)
Comment (NachoAlesLopez) [reply sale_mrp_link/__manifest__.py]: Done!

**Pull request merged**